documentation/src/main/wiki/LinuxAndWindowsSetup.md

```diff
@@ -17,30 +17,71 @@ Please note that full list of required tools and its versions can be found in [S
     * Configure site for the following path: `/opt/aet/apache/aet_reports/current`
     
 ##### Windows Setup
-1. Install Java 7 JDK and update JAVA_HOME environment variable.
-2. Change default console resolution - install VNC server (e.g. http://www.tightvnc.com/), connect by VNC client to console and change resolution (min. 1024x768).
-3. Turn off Windows Firewall (both, private and public network location settings).
-4. Install Karaf in version 2.3.9
-    * Update Apache Felix Framework to version 4.2.1
-    * Install Karaf as a Windows service
-    * Check if it's working under http://localhost:8181/system/console/ and credentials karaf/karaf.
-5. Install Browsermob in version 2.0.0
-    * Install Browsermob as a Windows service
-    * Check if it's working under http://localhost:9272/proxy.
-6. Install Firefox 38.6.0 ESR
-    * Choose 'Custom Install' and **do not install maintenance service** - this is very important
-    * Turn off automatic updates
-7. Check the following connections between Windows and Linux:
-    * MongoDB: `telnet ${LINUX_MACHINE_PRIVATE_IP} 27017`
-    * ActiveMQ: `telnet ${LINUX_MACHINE_PRIVATE_IP} 61616`
-    * ActiveMQ's JMX: `jconsole.exe ${LINUX_MACHINE_PRIVATE_IP}:11199`
+Windows Setup.
+1. Follow to link and download all package to your local instance.
+2. Turn off Windows Firewall (both, private and public network location settings).
+3. Install Notepadd++.
```

Comment:
Could we move Notepad++ as well as Baretail into some sectiion called like **Optional Software** or *Additional** or something like that?